### PR TITLE
feature/core-bluetooth-dispatch-queue

### PR DESCRIPTION
**Pull request by tueric**

### CODE REVIEW

File: docs/docsets/UB.docset/Contents/Resources/Documents/undocumented.json [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
     {
       "file": "/Users/deaneigenmann/Documents/ub/UB.swift/Sources/UB/Node.swift",
       "line": 14,
       "symbol": "Node.init()",
       "symbol_kind": "source.lang.swift.decl.function.method.instance",
       "warning": "undocumented"
     }
+=======
+
+>>>>>>> master
```
Comment (decanus) [inline docs/docsets/UB.docset/Contents/Resources/Documents/undocumented.json]: ? @tueric, maybe delete docs/ and regenerate?

File: docs/docsets/UB.docset/Contents/Resources/Documents/badge.svg [hunk 2]
```diff
@@ -18,11 +18,19 @@
...
     <text x="1095" y="140" transform="scale(.1)" textLength="250">
-      96%
+      97%
+=======
+    <text x="1135" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="330">
+      100%
+    </text>
+    <text x="1135" y="140" transform="scale(.1)" textLength="330">
+      100%
+>>>>>>> master
```
Comment (decanus) [inline docs/docsets/UB.docset/Contents/Resources/Documents/badge.svg]: merge

File: Sources/UB/Transports/CoreBluetoothTransport.swift [hunk 3]
```diff
@@ -112,8 +115,9 @@ extension CoreBluetoothTransport: CBPeripheralManagerDelegate {
                 // @todo
                 return
             }
-
-            delegate?.transport(self, didReceiveData: data, from: Addr(request.central.identifier.bytes))
+            DispatchQueue.main.async { () -> Void in
```
Comment (decanus) [inline Sources/UB/Transports/CoreBluetoothTransport.swift]: Add newline before statement and after

File: docs/docsets/UB.docset/Contents/Resources/Documents/undocumented.json (hunk 1 above)
Comment (tueric) [reply docs/docsets/UB.docset/Contents/Resources/Documents/undocumented.json]: ok

File: Sources/UB/Transports/CoreBluetoothTransport.swift [hunk 4]
```diff
@@ -12,6 +12,9 @@ public class CoreBluetoothTransport: NSObject {
     private let centralManager: CBCentralManager
     private let peripheralManager: CBPeripheralManager
 
+    private static let centralQueue = DispatchQueue(label: "com.UB.centralQueue", attributes: .concurrent)
```
Comment (decanus) [inline Sources/UB/Transports/CoreBluetoothTransport.swift]:
```suggestion
    private static let centralQueue = DispatchQueue(label: "com.ultralight-beam.bluetooth.centralQueue", attributes: .concurrent)
```

File: Sources/UB/Transports/CoreBluetoothTransport.swift [hunk 5]
```diff
@@ -12,6 +12,9 @@ public class CoreBluetoothTransport: NSObject {
...
+    private static let peripheralQueue = DispatchQueue(label: "com.UB.peripheralQueue", attributes: .concurrent)
```
Comment (decanus) [inline Sources/UB/Transports/CoreBluetoothTransport.swift]:
```suggestion
    private static let peripheralQueue = DispatchQueue(label: "com.ultralight-beam.bluetooth.peripheralQueue", attributes: .concurrent)
```

**Pull request merged**